### DISCUSSION TITLE
Airflow gif

### DISCUSSION THREAD

**Discussion by water-escape** [re-1st/HA-3d-floorplan#4] [General]
@re-1st  the airflow gif is not transparent. How did you make it transparent?
When i pull it online through an background remover tool, that works, but not in HA. It just plays it 1 time then.

**Comment by re-1st** [comment]
hi there,
no need to use a background remover. Use "mix-blend-mode : lighten"  in the style of the card. Like this:


style:
  mix-blend-mode: lighten

**Comment by water-escape** [comment]
@re-1st dankjewel.
This works, didnt know that one. However i need to rotate the image , after i did that in a tool, it just plays it one time again. Seems like i can not edit the file.

EDIT: rotate works with   transform: translateX(-50%) rotate(45deg)

Another question: Do you also have one with the red color? (red for heat and blue for cool)

**Reply by water-escape** [reply]
@re-1st where did you get that gif? i would like to have it in red as well.

**Reply by re-1st** [reply]
> @re-1st where did you get that gif? i would like to have it in red as well.

oof i really don't remember. great idea though! 
i think it might work if you add this 
style:
  filter: hue-rotate(110deg)

not sure about that number so maybe it is some degrees off

let me know if that works

**Comment by water-escape** [comment]
Haha nice, this works, but it’s a bit of pink color. So need to play with the degrees.





From: Smartstyler ***@***.***>
Date: Friday, 20 December 2024 at 13:53
To: re-1st/HA-3d-floorplan ***@***.***>
Cc: water-escape ***@***.***>, Author ***@***.***>
Subject: Re: [re-1st/HA-3d-floorplan] Airflow gif (Discussion #4)

@re-1st<https://github.com/re-1st> where did you get that gif? i would like to have it in red as well.

oof i really don't remember. great idea though!
i think it might work if you add this
style:
filter: hue-rotate(110deg)

not sure about that number so maybe it is some degrees off

let me know if that works
